tests/antithesis/README.md

```diff
@@ -8,6 +8,7 @@ Run this command from the `antithesis/test-template` directory:
 
 ```bash
 make antithesis-build-client-docker-image
```

Comment:
We need to change how the image is build in antithesis workflow.

https://github.com/etcd-io/etcd/blob/f206a8a579eb121d17e314637cb3fd90b8e30cf1/.github/workflows/antithesis-test.yml#L75-L83

We could replace the docker/build-push-action with just invoking makefile, however we would also need to retag and push the images to `us-central1-docker.pkg.dev/molten-verve-216720/linuxfoundation-repository/etcd-server:latest`

Reply:
Thank you.  Sorry for some reasons it was still not registered in my brain that we have the workflow files too... Should be the last time I forget about it.